camera_calibration/src/camera_calibration/camera_calibrator.py

```diff
@@ -139,7 +139,7 @@ class CalibrationNode:
     def __init__(self, boards, service_check = True, synchronizer = message_filters.TimeSynchronizer, flags = 0,
                 fisheye_flags = 0, pattern=Patterns.Chessboard, camera_name='', checkerboard_flags = 0,
                 max_chessboard_speed = -1, queue_size = 1, no_gui = False, no_mono_if_stereo = False,
-                mask = False):
+                mask=False):
```

Comment:
I would avoid making these linting changes if they are not done on a whole file... then diff to the upstream will be less